kube/manifests/prometheus-gcs.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: prometheus-gcs
+  labels:
+    app: prometheus-gcs
+    thanos-peer: "true"
+spec:
+  serviceName: "prometheus"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prometheus-gcs
+      thanos-peer: "true"
+  template:
+    metadata:
+      labels:
+        app: prometheus-gcs
+        thanos-peer: "true"
+    spec:
+      containers:
+      - name: prometheus
+        image: quay.io/prometheus/prometheus:v2.0.0
+        args:
+        - "--storage.tsdb.retention=24h"
+        - "--config.file=/etc/prometheus/prometheus.yml"
+        - "--storage.tsdb.path=/var/prometheus"
+        - "--storage.tsdb.min-block-duration=2h"
+        - "--storage.tsdb.max-block-duration=2h"
+        ports:
+        - name: http
+          containerPort: 9090
+        volumeMounts:
+        - name: config
+          mountPath: /etc/prometheus
+        - name: data
+          mountPath: /var/prometheus
+      - name: thanos-sidecar
+        image: improbable/thanos:latest
```

Comment:
i take it this is there now and were not using Barteks docker hub 👍

Reply:
yea!